gc_little_helper_II.user.js

```diff
@@ -17385,6 +17446,117 @@ var mainGC = function() {
                 $("#ShowHomezoneCircles").toggle();
             });
 
+            // Custom links for cache listing:
+            // -------------------------------
+            // Fill options for select field of cache type.
+            function createOptionsForCachetype_ilinks(listing, cachetype) {
```

Comment:
(Changes has to be made in init.js)

APE Caches aren't events

IMHO the name "Geocaching HQ Block Party" can be confusing (even GS use the term [here](https://www.geocaching.com/about/cache_types.aspx)), since this year Block parties take place all over the world. Not just at the HQ.